vme/src/dilrun.cpp

```diff
@@ -1010,7 +1015,16 @@ static int check_interrupt(dilprg *prg)
                 (prg)->fp->pc++;
                 (prg)->fp->tmpl->nInstructions++;
                 assert(prg->fp->pc <= &(prg->fp->tmpl->core[prg->fp->tmpl->coresz]));
-                (g_dilfe_func[*(prg->fp->pc - 1)].func(prg));
+                try
+                {
+                    auto dil_function = g_dilfe_func.at(*(prg->fp->pc - 1));
```

Comment:
Ok I see how the map works now. Are you sure that this won't lead to a measurable performance degradation? there's a really serious amount of DILs being called. Before the time for this part was O(1) and now it's O(log(n)). 

Looking at wstat dil on the server, in 22 hours 100s of millions of DIL programs are called. I'd guess more like 1B+ times for this particular function.

Do you have a way to profile? Then you could run the MUD with all zones, login, do a wtime 7200 to boost time. Once it's done, shutdown. See the profile data. Once for each ofc. What do you think?

Reply:
Absolutely, I will  profile it and get before and afters. With 200+ items I really doubt its even going to show up in the profiles but proving it is better.

As a side thing, if I load all the zones - my tests with wtime timewarp don't run anywhere near as fast as it used too with just the project zones. Is that expected, because of the increase in the zones.